docs/processors/builtin/error.mdx

```diff
@@ -0,0 +1,134 @@
+---
+IMPORTANT: This file was generated using src/processorgen/main.go. DO NOT EDIT.
+
+title: 'error'
+sidebar_position: 5
+---
+
+import ReactDiffViewer from 'react-diff-viewer';
+import Chip from '@mui/material/Chip';
+import Box from "@mui/system/Box";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# `error`
+
+Returns an error for all records that get passed to the processor.
+
+## Description
+
+Any time a record is passed to this processor it returns an error,
+which results in the record being sent to the DLQ if it's configured, or the pipeline stopping.
+
+**Important:** Make sure to add a [condition](https://conduit.io/docs/processors/conditions)
+to this processor, otherwise all records will provoke an error.
```

Comment:
Only leaving this suggestion since I left it on the source as well https://github.com/ConduitIO/conduit/pull/1598.

```suggestion
to this processor, otherwise all records will trigger an error.
```